docs/src/reference/exporters.md

```diff
@@ -0,0 +1,96 @@
+> define different exporters, then these will share the same class loader.
+>
+> This has some nice properties, primarily that different exporters can depend on
+> the same third party libraries without having to worry about versions, or class
+> name collisions.
+>
+> Additionally, exporters use the system class loader for system classes, or
+> classes packaged as part of the Zeebe JAR.
+
+Exporter specific configuration is handled through the exporter's `[exporters.args]`
+nested map. This provides a simple `Map<String, Object>` which is passed directly
+in form of a [Configuration](https://github.com/zeebe-io/zeebe/tree/{{commit}}/exporter-api/src/main/java/io/zeebe/exporter/context/Configuration.java)
+object when the broker calls the `Exporter#configure(Configuration)` method.
+
+Configuration occurs at two different phases: during the broker startup phase, and
+once every time a leader is elected for a partition.
+
+## Processing
+
+As mentioned before in this guide, at any given point, there is exactly one leader
+node for a given partition. Whenever a node becomes the leader for a partition, one
+of the things it will do is run an instance of an
+[exporter stream processor](https://github.com/zeebe-io/zeebe/tree/{{commit}}/broker-core/src/main/java/io/zeebe/broker/exporter/stream/ExporterStreamProcessor.java).
+
+This stream processor will create exactly one instance of each configured exporter,
+and forward every record written on the stream to each of these in turn.
+
+> **Note:** this implies that there will be exactly one instance of every exporter for
+> every partition: if you have 4 partitions, and at least 4 threads for processing,
+> then there are potentially 4 instances of your exporter exporting simultaneously.
+
+Note that Zeebe only guarantees at-least-once semantics, that is, a record will be
+seen at least once by an exporter, and maybe more. Cases where this may happen
+include:
+
+* During reprocessing after raft failover (i.e. new leader election)
+* On error if the position has not been updated yet
+
+To reduce the amount of duplicate records an exporter will process, the stream
+processor will keep track of the position of the last successfully exported record
+for every single exporter; the position is sufficient since a stream is an ordered
+sequence of records whose position is monotonically increasing. This position is
+set by the exporter itself once it can guarantee a record has been successfully
+updated.
+
+> **Note:** although Zeebe tries to reduce the amount of duplicate records an
+> exporter has to handle, it is likely that it will have to; therefore, it is
+> necessary that export operations be idempotent. This can be implemented either in
+> the exporter itself, but if it exports to an external system, it is recommended
+> that you perform deduplication there to reduce the load on Zeebe itself. Refer
+> to the exporter specific documentation for how this is meant to be achieved.
```

Comment:
I find the last sentence confusing, as I am already reading exporter-specific documentation here :) Also, it makes assumptions how people document their own code.

Maybe we can rather assume that the user who configures the exporter is the same user who implements it (or at least has the end-to-end knowledge). Then we could simply drop this sentence.

Reply:
To remove